tests/unit/lms/validation/_lti_launch_params_test.py

```diff
@@ -62,7 +83,10 @@ def set_url_(url):
     def schema(self, pyramid_request):
         return LaunchParamsURLConfiguredSchema(pyramid_request)
 
-    @pytest.fixture
-    def pyramid_request(self, pyramid_request):
-        pyramid_request.content_type = "application/x-www-form-urlencoded"
-        return pyramid_request
+
+@pytest.fixture
+def pyramid_request(pyramid_request):
+    pyramid_request.content_type = "application/x-www-form-urlencoded"
+    pyramid_request.params.update({"resource_link_id": "DUMMY-LINK"})
```

Comment:
Due to the addition of this requirement, we now need this for both schema to work